fs/io_zenfs.cc

```diff
@@ -636,6 +637,72 @@ size_t ZonedRandomAccessFile::GetUniqueId(char* id, size_t max_size) const {
   return zoneFile_->GetUniqueId(id, max_size);
 }
 
+ZenFSGCWorker::ZenFSGCWorker() {
+  
+  total_residue_ = 0;
+
+}
+
+void ZenFSGCWorker::CheckZoneValidResidualData() {
+
+  std::map<std::string, ZoneFile*>::iterator it;
+  fs->files_mtx_.lock();
+  for(it = fs->files_.begin(); it != fs->files_.end(); it++) {
+
+    ZoneFile* existFile;
+    existFile = it->second;
+
+    for(auto ext_it : existFile->extents_) {
+
+      ZoneExtent* extent;
+      extent = ext_it;
```

Comment:
It seems that `extent` and `ext_it` are the same thing. Should we use a single identifier for both of them?

Reply:
The code may not be the latest. I did some modifications to it.